org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/JavaEclipseCompiler.scala

```diff
@@ -23,7 +22,7 @@ class JavaEclipseCompiler(p: IProject, monitor: SubMonitor) extends JavaCompiler
 
   override def project = p
 
-  override def compile(sources: Array[File], classpath: Array[File], output: Output, options: Array[String], log: Logger): Unit = {
+  override def run(sources: Array[File], options: Array[String], reporter: Reporter, log: Logger): Boolean = {
```

Comment:
not sure about this but I think would be fine to know from method signature that argument is not used like
`override def run(sources: Array[File], notUsedOptions: Array[String], notUsedReporter: Reporter, notUsedLog: Logger): Boolean ` (I hope I didn't make a mistake here)